apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -0,0 +1,392 @@
+import React, {useEffect, useState, useRef} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import classnames from 'classnames';
+import style from './rubrics.module.scss';
+import {
+  learningGoalShape,
+  reportingDataShape,
+  studentLevelInfoShape,
+  submittedEvaluationShape,
+  aiEvaluationShape,
+} from './rubricShapes';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import {
+  BodyThreeText,
+  BodyFourText,
+  ExtraStrongText,
+  StrongText,
+  Heading6,
+} from '@cdo/apps/componentLibrary/typography';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import EvidenceLevels from './EvidenceLevels';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import AiAssessment from './AiAssessment';
+import HttpClient from '@cdo/apps/util/HttpClient';
+import {UNDERSTANDING_LEVEL_STRINGS} from './rubricHelpers';
+
+const invalidUnderstanding = -1;
+
+export default function LearningGoals({
```

Comment:
I named it LearningGoals because it is a component for displaying all of the learning goals in a rubric. I could go with LearningGoalContainer if you think that's more descriptive.

Reply:
Sorry, I missed that it displays multiple learning goals when I made that comment! LearningGoals sounds good, then